src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -174,55 +240,85 @@
       "defaultValue": "true",
       "description": "Includes the UI test project"
     },
-    "configuration": {
+    "server": {
+      "displayName": "Server - API",
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "true",
+      "description": "Includes a Server project for an API & host for the WebAssembly project"
+    },
+    "use-configuration": {
```

Comment:
We need to have consistency with the naming of these symbols. My preference is to drop the "use-", or perhaps only have the "use-" where the symbol is computed.